queue_job/jobrunner/channels.py

```diff
@@ -692,10 +748,11 @@ class ChannelManager(object):
     >>> cm.get_wakeup_time()
     104
 
-    Let's test throttling in combination with a queue reaching full capacity.
+    Let's test throttling in combination with a queue reaching full capacity
+    (but without the root channel having more capacity available)
 
     >>> cm = ChannelManager()
-    >>> cm.simple_configure('root:4,T:2:throttle=2')
+    >>> cm.simple_configure('root:2,T:2:throttle=2')
```

Comment:
This change (as well as the change in the test below) looks suspicious. Does the combination of this 'native' throttling need special care in combination with overcapacity?

Reply:
I've adapted the code to ignore throttled channels.